packages/repository-json-schema/src/build-schema.ts

```diff
@@ -203,6 +204,31 @@ export function metaToJsonProperty(meta: PropertyDefinition): JSONSchema {
   return result;
 }
 
+/**
+ * Checks and return property schema definition for the relation
+ * @param relMeta Relation metadata object
+ * @param targetRef Schema definition for the target model
+ */
+export function getPropDefForRelatedModel(
```

Comment:
We prefer to use full names instead of abbreviations.

How about the following?

```ts
export function getNavigationalPropertyForRelation(
  relationMeta: RelationMetadata,
  targetSchema: JSONSchema
): JSONSchema {
  // ...
}
```

Reply:
Yeah. That's better. Changing it. Will rebase and squash as well.